README.md

```diff
@@ -1006,6 +1006,18 @@ in inheritance.
     end
     ```
 
+* When defining ActiveRecord model scopes, wrap the relation in a `lambda`.  A
+  naked relation forces a database connection to be established at class load
+  time (instance startup).
+
+    ```ruby
+    # bad
+    scope :foo, where(:bar => 1)
```

Comment:
@romanfuchs I'm guessing the majority of those like you have in your example are older.  Most newer code in our codbase prefers the use of `where` blocks to `:conditions`